src/fidesops/ops/email_templates/templates/subject_identity_verification.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>ID Code</title>
+</head>
+<body>
+<main>
+    <p>
+        Your privacy request verification code is {{code}}.
+        Please return to the Privacy Center and enter the code to
+        continue. This code will expire in {{minutes}} minutes
```

Comment:
It is possible that `{{minutes}}` is 0. What do we do then? I know that is outside the scope of this ticket though.

Reply:
It is possible. The solution would probably be to embed an if/else block within the template. Jinja has some pretty advanced templating options: [https://jinja.palletsprojects.com/en/3.1.x/templates/#if](https://jinja.palletsprojects.com/en/3.1.x/templates/#if)

The default timeout is [10 minutes](https://github.com/ethyca/fidesops/blob/main/src/fidesops/ops/core/config.py#L60). A user could set it the ttl to `<60` seconds. That would hopefully be pretty rare though. It seems silly to send verification emails that would say `This code will expire in 33 seconds` 😂 

Maybe we can set a minimum ttl for verification codes?